apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-logs-sidebar.tsx

```diff
@@ -1,3 +1,71 @@
-export const TestWorkflowLogsSidebar = () => {
-  return <aside className="flex h-full w-[500px] flex-col border-l">Logs</aside>;
+import { ActivityPanel } from '@/components/activity/activity-panel';
+import { useEffect, useState } from 'react';
+import { JobStatusEnum } from '@novu/shared';
+import { Loader2 } from 'lucide-react';
+import { WorkflowTriggerInboxIllustration } from '../../icons/workflow-trigger-inbox';
+import { useFetchActivities } from '../../../hooks/use-fetch-activities';
+
+interface TestWorkflowLogsSidebarProps {
```

Comment:
Nit picky one: If not mistaken for react component props we prefer types in the new Dashboard